package.json

```diff
@@ -49,7 +49,7 @@
     "gulp-sass": "5.0.0",
     "gulp-sass-lint": "1.4.0",
     "gulp-uglify": "3.0.2",
-    "jest": "29.0.1",
+    "jest": "^29.0.1",
```

Comment:
Is there a reason we've made this a little more strict about version updates?  If we're changing this would it be worth bringing us up to date with the current version 29.7?